source/javascript/src/utils/Router.js

```diff
@@ -1,4 +1,4 @@
-const prod = true
+const prod = false
```

Comment:
I'm not sure whether this should be kept `true` or `false`. I think when testing locally it should be `false`, but when pushing onto the main branch it should be `true`? @yuzilyu @NoahT

Reply:
yeah if it's false the actual website's routing won't work

Reply:
I think we could merge it and change this line of code directly on github